app/schema/questionnaire.py

```diff
@@ -28,10 +28,13 @@ def add_group(self, group):
             group.container = self
 
     def get_item_by_id(self, item_id):
+        answer_without_instance_id = item_id.split('_')[0]
```

Comment:
Does this need to use the ```_extract_answer_instance_id(...)``` method; this simple split is likely to give false positives etc.